client/packages/invoices/src/Prescriptions/DetailView/SidePanel/PrescriptionDetailsSection.tsx

```diff
@@ -73,26 +28,17 @@ export const PrescriptionDetailsSectionComponent: FC = () => {
     theirReferenceInput,
   ]);
 
-  if (!createdDatetime) return null;
-
   useEffect(() => {
     if (!data) return;
     const { theirReference } = data;
     setTheirReferenceInput(theirReference);
   }, [data]);
 
+  if (!createdDatetime) return null;
```

Comment:
Moved underneath useEffect, eslint putting out an error that the useEffect can't be rendered conditionally.